# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/shell/nsWindowsShellService.cpp
# Commit: 49f09ae5e13a
# Full Hash: 49f09ae5e13a0fb94508c7e7ca43f9e9f9aa8fd5
# Author: Ben Hearsum <bhearsum@mozilla.com>
# Date: 2022-04-14 21:45:12
# Regressor Bug: 1763573
# File Overlap Count: 1
# Description:
#   Bug 1763573: temporarily disable Windows 7 taskbar pinning to determine whether or not it's causing crashes r=mhowell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D143719
# ==============================================================================

diff -r 23a12c0ddc69 -r 49f09ae5e13a browser/components/shell/nsWindowsShellService.cpp
--- a/browser/components/shell/nsWindowsShellService.cpp	Thu Apr 14 16:22:21 2022 +0000
+++ b/browser/components/shell/nsWindowsShellService.cpp	Thu Apr 14 16:26:34 2022 +0000
@@ -1042,6 +1042,13 @@
 }
 static nsresult PinCurrentAppToTaskbarWin7(bool aCheckOnly,
                                            nsAutoString aShortcutPath) {
+  // Currently disabled due to
+  // https://bugzilla.mozilla.org/show_bug.cgi?id=1763573 At the time of
+  // writing, the only callers eat any errors from here so it's safe to return
+  // one. This is a short term workaround to confirm that this code is indeed
+  // causing the bug above. If it is, we'll rework this code to avoid it.
+  return NS_ERROR_ABORT;
+
   // This is a less generalized version of the code from the NSIS
   // Invoke Shell Verb plugin.
   // https://nsis.sourceforge.io/Invoke_Shell_Verb_plugin
